@navikt/core/react/src/form/file-upload/FileUpload.tsx

```diff
@@ -189,19 +192,11 @@ export const FileUpload = forwardRef<HTMLDivElement, FileUploadProps>(
           </div>
         </div>
       </FileUploadContext.Provider>
-    )
+    );
   }
 ) as FileUploadComponent;
 
-export interface FileUploadComponent
```

Comment:
Flyttet denne til toppen av filen, under props for å matche lignende kode i systemet